packages/graphql/server/tests/mutators/callbacks.test.ts

```diff
@@ -28,8 +28,14 @@ const defaultModelOptions = {
 const Foo = createGraphqlModel({
   ...defaultModelOptions,
   graphql: { typeName: "Foo", multiTypeName: "Foos" },
+  permissions: {
```

Comment:
Ok but don't repeat that throughout the test, create a "memberPermissions" variable and reuse it in relevant tests